database/database.go

```diff
@@ -0,0 +1,38 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	_ "github.com/lib/pq"
+)
+
+const (
+	host     = "localhost"
+	port     = 5432
+	user     = "dummyuser"
+	password = "12345678"
+	dbname   = "dummydb"
```

Comment:
A better way to store the credentials is in the environment variable. As for reading it, you can use [caarlos0/env library](https://github.com/caarlos0/env) to read environment variables into Golang variables.

Reply:
As for loading the `.env` file (for local testing), you can [try using Docker Compose](https://docs.docker.com/compose/environment-variables/set-environment-variables/#additional-information).

Reply:
@sungindra For loading `.env` file, if you don't want to try Docker Compose, the other alternative is to use [godotenv](https://github.com/joho/godotenv).